setup.py

```diff
@@ -39,6 +39,7 @@
         'taca.subcommands': [
             'storage = taca.storage.cli:storage',
             'analysis = taca.analysis.cli:analysis',
+            'delivery = taca.utils.cli:delivery',
```

Comment:
how about `monitor` or `run-monitor` ? :eyes: 
(I am not good with names :sweat_smile: )